src/cpp/desktop/resources/freedesktop/rstudio.xml

```diff
@@ -45,6 +45,12 @@
     <glob pattern="*.rpres"/>
   </mime-type>
 
+  <mime-type type="text/x-r-notebook">
```

Comment:
Does it make sense to define a mime-type here? It looks like this helps RStudio get priority in opening `.nb.html` files + might be needed for icons on Linux; however, we might also want to consult https://specifications.freedesktop.org/shared-mime-info-spec/shared-mime-info-spec-0.18.html to see if there's a better way to declare this.